### DISCUSSION TITLE
Openstreetmap Integration

### DISCUSSION THREAD

**Discussion by Simola4theWicked** [nocodb/nocodb#7578] [Q&A]
Hello,

I am working on a dataset for several different sign types on a hiking track.
I am planning on using NocoDB. The datasets for each type will include stuff like, condition of the sign, a picture of the sign, name of the track, location and so on.

I was wondering if there is a way to show all these points on a openstreetmap based on the geodata.
Ideally you could click on the pints on the map and you will see the other datapoints.

I want to do this for 9 hiking tracks in the same area.

**Comment by kamazeuci** [comment]
Did you manage to get your project going? I am also interested in integrating nocodb and OSM, maybe through Leaflet

**Reply by Simola4theWicked** [reply]
Not yet. Still waiting for the update thatwill bring the feature.
